app/DashBoard/Controls/CreateTestServerDataGrid/Cells.latte

```diff
@@ -0,0 +1,21 @@
+{define col-repository}
+	<td>
+		{$cell->name}
+	</td>
+{/define}
+
+{define col-commit}
+	<td>
+		{$cell|truncate:7,''}
+	</td>
+{/define}
+
+{define col-finish}
+	<td>
+		{$cell|date:'j. n. Y H:i:s'}
```

Comment:
Filtr pro výpis datumu jsem už použil ve dvou komponentách. Už by to stálo za přidání do globálních filtrů, aby byly všude